skepticoin/networking/peer.py

```diff
@@ -226,7 +208,10 @@ def step(self, current_time):
         # timeout is only implemented half-baked: it currently only limits when an new GetBlocksMessage will be sent;
         # but doesn't take the timed-out peer out of the loop that it's already in. This is not necessarily a bad thing.
 
-    def should_actively_fetch_blocks(self, current_time):
+    def should_actively_fetch_blocks(self, current_time: int) -> bool:
+
+        assert self.coinstate
```

Comment:
I don't like this one; I'm assuming it's needed to get mypy to shut up about something, but I'd think it should be possible to do something more idiomatic instead.

Reply:
Resolving this one, see other thread about this topic.